browser/main/modals/PreferencesModal/Crowdfunding.js

```diff
@@ -23,18 +23,18 @@ class Crowdfunding extends React.Component {
     return (
       <div styleName='root'>
         <div styleName='header'>{i18n.__('Crowdfunding')}</div>
-        <p>{i18n.__('Dear everyone,')}</p>
+        <p>{i18n.__('Dear Boostnote users,')}</p>
         <br />
         <p>{i18n.__('Thank you for using Boostnote!')}</p>
         <p>{i18n.__('Boostnote is used in about 200 different countries and regions by an awesome community of developers.')}</p>
         <br />
-        <p>{i18n.__('To continue supporting this growth, and to satisfy community expectations,')}</p>
+        <p>{i18n.__('To support our growing userbase, and satisfy community expectations,')}</p>
         <p>{i18n.__('we would like to invest more time and resources in this project.')}</p>
         <br />
-        <p>{i18n.__('If you like this project and see its potential, you can help by supporting us on OpenCollective!')}</p>
+        <p>{i18n.__(`If you use Boostnote and see it's potential, help us out by supporting the project on OpenCollective!`)}</p>
```

Comment:
I think you should leave it as `its`. English is my second language so please correct me if I'm wrong.
Reference: https://en.oxforddictionaries.com/usage/its-or-it-s

Reply:
You are right! Good catch :)